.github/workflows/ci.yml

```diff
@@ -261,6 +228,77 @@ jobs:
           name: flv-test-${{ matrix.target }}
           path: target/${{ matrix.target }}/release/flv-test
 
+
+  run_platform_unit_test:
```

Comment:
Can we use a build matrix rather than copying this job?

Reply:
Unfortunately, it mislead to wrong job name.    Instead, let's try to use custom action